tests/test_classification.py

```diff
@@ -105,6 +118,52 @@ def test_rp(sparse):
     assert True
 
 
+@pytest.mark.filterwarnings("ignore::UserWarning")
+def test_rare_label():
+    data = make_rare_label(DATA)
+    test_rp(data)
+
+
+def test_invalid_inputs():
+    data = make_data(sparse=False, sizes=[1, 1, 1])
+    try:
+        test_rp(data)
+    except Exception as e:
+        assert "Need more data" in str(e)
+    try:
+        rp = LearningWithNoisyLabels(
+            clf=LogisticRegression(multi_class="auto", solver="lbfgs", random_state=SEED)
+        )
+        rp.fit(
+            data["X_train"],
+            data["labels"],
+            find_label_issues_kwargs={"return_indices_ranked_by": "self_confidence"},
+        )
+    except Exception as e:
+        assert "not supported" in str(e)
+
```

Comment:
Same as above, add else to ensure that exception is thrown

Reply:
good catch, added